apps/app/src/components/Page/CopyDropdown.jsx

```diff
@@ -118,7 +118,7 @@ const CopyDropdown = (props) => {
           <span id={dropdownToggleId}>{children}</span>
         </DropdownToggle>
 
-        <DropdownMenu positionFixed modifiers={{ preventOverflow: { boundariesElement: 'viewport' } }}>
+        <DropdownMenu positionFixed modifiers={{ preventOverflow: { boundariesElement: 'viewport' } }} style={{ zIndex: 1055 }}>
```

Comment:
Is `style={{ zIndex: 1055 }}` necessary?

Reply:
@jam411 
I see a reference to a PageItemControl using that style. I think it's necessary to add those styles to the CopyDropdown because when it opens then we scroll up, the CopyDropdown will enter below the Sub Navigation